charts/airbyte/Chart.yaml.test

```diff
@@ -0,0 +1,64 @@
+apiVersion: v2
```

Comment:
why do we need a `Chart.yaml.test`?

Reply:
Ah I see - we are using this in the test script

Reply:
Local testing + acceptance testing